pandas/tests/extension/base/constructors.py

```diff
@@ -64,6 +64,15 @@ def test_from_dtype(self, data):
         result = pd.Series(list(data), dtype=str(dtype))
         self.assert_series_equal(result, expected)
 
+        # gh-30280
+
+        expected = pd.DataFrame(data).astype(dtype)
```

Comment:
Can this test be parametrized instead? Or is the `.astype` call a hanging point?

Reply:
> Can this test be parametrized instead?

maybe could parametrize in a follow-on if this test is OK.

> Or is the `.astype` call a hanging point?

and also the assert method changes, so the four tests could end up with less clarity. maybe could split the test, but that would result in additional skips required in _numpy and json tests.